<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FailureWidget</class>
 <widget class="QWidget" name="FailureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbox_select_all">
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_checked_count">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_expand">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_new_plan">
       <property name="text">
        <string>新建PLAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_solution">
       <property name="text">
        <string>查看分析</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="result_tree_widget">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
